test/src/main/java/io/strimzi/test/executor/Exec.java

```diff
@@ -339,6 +340,11 @@ private void storeOutputsToFile() {
      * @return true.false
      */
     public static boolean isExecutableOnPath(String cmd) {
+        var osName = System.getProperty("os.name");
+        if (osName.toLowerCase(Locale.US).startsWith("windows")) {
+            cmd += ".exe";
+        }
```

Comment:
Does this work well on Windows with the Linux subsystem?

Reply:
Yes. I couldn't run tests in IntelliJ without it and that made this all a lot more difficult.

Reply:
I mean -> most contributors from Windows seem to use the Linux subsystem. But I have no idea if that identifies itself as Linux and won't be affected by this condition, or if that uses the `.exe` extension as well or how does it work on Windows. Just want to make sure this will not break anything.

Reply:
Java running in WSL just sees that he is living in Linux of some sort. `os.name` only returns as `windows-blah-blah` with running in windows, not from WSL. The problem is that it is checking if for exemple `kubectl` exists and has execute permissions. On windows you can execute `kubectl`, but `kubectl` doesn't exist; `kubectl.exe` exists. They run in WSL because they have no other choice. Running maven for every change is a lot more time consuming. That check for permissions without correcting the name means that running the tests in windows fails with a message that it can't find a cluster. This change doesn't change how it is executed, it just checks the correct file name on windows and Linux.

Reply:
Great, thanks.